services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -2612,7 +2612,14 @@ public Version incrementVersionIdempotent(
       Optional<X509Certificate> requesterCert,
       long rewindTimeInSecondsOverride,
       Optional<String> emergencySourceRegion,
-      boolean versionSwapDeferred) {
+      boolean versionSwapDeferred,
+      boolean canaryRegionPush) {
+    if (canaryRegionPush) {
+      // only parent controller should handle this request. See @{link
+      // VeniceParentHelixAdmin#incrementVersionIdempotent}
```

Comment:
use /** xxx */ to access the link easily?

Reply:
Also, I believe it should be `{@link ...}` instead of `@{link}`

Reply:
Updated. Please review again.